### ISSUE TITLE
Small bug (I think) in examples/ directory, at the file called draw_text.rb

### ISSUE THREAD

**Issue by rubyFeedback** [kojix2/LibUI#39]
Heya kojix2,

I noticed you extended the examples, so I went on to test things. :D

Unfortunately it fails to me as superuser:

    ruby draw_text.rb 

The error I get is:

```
/root/.gem/gems/libui-0.0.12/lib/libui/ffi.rb:22:in `call': no implicit conversion of String into Integer (TypeError)
        from /root/.gem/gems/libui-0.0.12/lib/libui/ffi.rb:22:in `uiOpenTypeFeaturesAdd'
        from /root/.gem/gems/libui-0.0.12/lib/libui/libui_base.rb:39:in `public_send'
        from /root/.gem/gems/libui-0.0.12/lib/libui/libui_base.rb:39:in `block (2 levels) in <module:LibUIBase>'
        from draw_text.rb:66:in `make_attribute_string'
        from draw_text.rb:132:in `<main>'
```

I am not entirely sure where the error is (I find it not so easy to discover ffi-related errors in general) but I wanted to mention this nonetheless. Please disregard this message if it works for you (I use libui gem 0.0.12, but I used the git clone of the repository here, so not sure if that has anything to do with it or not; I mention it just in case). Thanks for reading!

**Comment by kojix2** [comment]
Hi!
Thanks for the report.
I had committed the changes related to this to the develop branch, but forgot to merge it into main.

Related to https://github.com/ruby/fiddle/issues/96

**Comment by kojix2** [comment]
Hi! 

Thank you for your report.
I've checked and this has been [fixed](https://github.com/kojix2/LibUI/commit/ed4f86bf70000b7278e071ff8dc98eb3dcc1ab3b) in the main branch.

```sh
git clone https://github.com/kojix2/LibUI
cd LibUI
rake vendor:all_x64
bundle install
bundle exec ruby example/draw_text.rb
```

---

Related to: https://github.com/ruby/fiddle/issues/96

In C, there is a difference between the char type and the string type. In Fiddle, char is uint8.
So when you pass a char to a function, you need to make it unit8. This is the same as the Crystal language. See the note on the following page.

https://crystal-lang.org/reference/syntax_and_semantics/c_bindings/fun.html#types-in-c-bindings

However, this specification is not always helpful, so I reported it to Fiddle. So it may be changed in the future.

**Comment by kojix2** [comment]
I have released v0.0.13 which includes this fix.
